ui/packages/consul-ui/lib/startup/templates/body.html.js

```diff
@@ -42,26 +42,28 @@ ${environment === 'production' ? `{{jsonEncode .}}` : JSON.stringify(config.oper
     "codemirror/mode/xml/xml.js": "${rootURL}assets/codemirror/mode/xml/xml.js"
   }
   </script>
-  <script data-app-name="${appName}" data-${appName}-services src="${rootURL}assets/consul-ui/services.js"></script>
+  <script src="${rootURL}assets/consul-ui/services.js"></script>
+  <script src="${rootURL}assets/consul-ui/routes.js"></script>
 ${
   environment === 'development' || environment === 'staging'
     ? `
-  <script data-app-name="${appName}" data-${appName}-services src="${rootURL}assets/consul-ui/services-debug.js"></script>
-  <script data-app-name="${appName}" data-${appName}-routing src="${rootURL}assets/consul-ui/routes-debug.js"></script>
+  <script src="${rootURL}assets/consul-ui/services-debug.js"></script>
+  <script src="${rootURL}assets/consul-ui/routes-debug.js"></script>
 `
     : ``
 }
 ${
   environment === 'production'
     ? `
 {{if .ACLsEnabled}}
-  <script data-app-name="${appName}" data-${appName}-routing src="${rootURL}assets/consul-acls/routes.js"></script>
+  <script src="${rootURL}assets/consul-acls/routes.js"></script>
 {{end}}
 {{if .PartitionsEnabled}}
-  <script data-app-name="${appName}" data-${appName}-routing src="${rootURL}assets/consul-partitions/routes.js"></script>
+  <script src="${rootURL}assets/consul-partitions/services.js"></script>
+  <script src="${rootURL}assets/consul-partitions/routes.js"></script>
 {{end}}
 {{if .NamespacesEnabled}}
-  <script data-app-name="${appName}" data-${appName}-routing src="${rootURL}assets/consul-nspaces/routes.js"></script>
+  <script src="${rootURL}assets/consul-nspaces/routes.js"></script>
```

Comment:
This is where we 'import' or don't 'import' additional configuration depending on what Consul says, things like `.NamespacesEnabled` are set to true/false directly by the backend (our index.html file is run through go-template before it gets served up to the user). Note: currently we only have injectable service/component for partitions currently - once this PR is merged we will be using the same approach for Namespaces and ACLs and more.